### PR TITLE
Fix: Collect and report coverage on PHP7.1 only

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] collects and reports coverage on PHP7.1 only

Follows #90.

💁‍♂️ Since Code Climate accepts the report once only, there's probably no need to collect and report it more than once, is there?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,9 +39,9 @@ before_script:
   - git config --global user.name "Travis CI"
 
 script:
-  - xdebug-enable
-  - vendor/bin/phpunit --coverage-clover=build/logs/clover.xml
-  - xdebug-disable
+  - if [[ "$WITH_COVERAGE" == "true" ]]; then xdebug-enable; fi
```
Comment (GordonDiggs) [inline .travis.yml]: Does this support multiline yml strings?

```yml
script: |
  if [[ "$WITH_COVERAGE" == "true" ]]; then
    xdebug-enable
    vendor/bin/phpunit --coverage-clover=build/logs/clover.xml
    xdebug-disable
  else 
    vendor/bin/phpunit
  fi
```

Comment (localheinz) [inline .travis.yml]: Let's find out!

Comment (localheinz) [inline .travis.yml]: It doesn't, but I extracted a function, see [`.travis/travis.sh`](https://github.com/codeclimate/php-test-reporter/pull/95/files#diff-02b12a62ed710ace978372c0e5250b01)!

**Comment by localheinz** [comment]
Thank you, @GordonDiggs!

**Pull request merged**